chihiro_todo/main.js

```diff
@@ -7,10 +7,18 @@ const doneArea = document.querySelector('#done_area');
 // taskをデータとして管理する変数
 let taskList = [];
 
+//担当者をデータとして管理する変数
+const personList = [{ id: "1", name: "柿崎"}, {id: "2", name: "柴田"}, {id: "3", name: "黒澤"}];
+
+//プルダウンで選択された担当者と一致するidをpersonListから探し出し、担当者名で返却する関数
+const getPersonNameById = (personId) => {
+    return personList.find(person => person.id === personId).name;
+}
+
 // タスクを追加する関数
-const addTask = (newTask) => {
+const addTask = function(newTask, selectPerson) {
```

Comment:
@chihiro-kakizaki 
```suggestion
const addTask = function(newTask, selectedPersonId) {
```
`選択された担当者ID`という意味で`selectedPersonId`と命名してあげた方が適切かと思いました！